debian/control-template

```diff
@@ -11,6 +11,8 @@ Build-Depends: debhelper (>= 9),
  texlive-latex-extra,
  texlive-plain-generic | texlive-generic-extra,
  latexmk
+ libmbedtls-dev
```

Comment:
This is the build-dependency.
Would we need a runtime dependency as well?

Reply:
I think we don't need to link the mbedtls explicitly. All targets are depending on 'libopen62541-<soname>' and this package in turn depends on '${shlibs:Depends}'. For my understanding, the runtime dependency should therefore be resolved automatically.

Reply:
FYI: is there a "comma" missing after latexmk ?

Reply:
You're right, thanks for the hint. ;)

Reply:
BTW: I agree with @jpfr that the binary section needs a dependency then.

IMHO the control section of the lib should look like
```
...
Package: libopen62541-<soname>
Architecture: any
Multi-Arch: same
Depends: ${shlibs:Depends}, ${misc:Depends}, libmbedtls10 | libmbedtls12
...
``` 
Without the binary dependency installation of the library will fail!

FYI `libmbedtls10 | libmbedtls12` is important to reflect the different names of the dependency package dependent on the Ubuntu/Debian version

Reply:
I have now update the PR according to your proposal @jrcoding. Thank's for your help :thumbsup: